train_prototype.py

```diff
@@ -116,6 +121,23 @@ def train(tasks, model, args, device):
                 iter_loss, acc))
             running_loss = 0.0
 
+        # evaluate in k shot fashion
+        if iterations % args.eval_every == 0:
+            test_model.load_state_dict(model.state_dict())
+            test_mean, _ = k_shot_testing(test_model, episodes, test_task, device, num_test_batches=args.num_test_batches)
```

Comment:
I think that might be helpful to log this in tensorboard too. What do you think about keeping track of both test_mean and test_std for all models?

Reply:
Totally makes sense we should do